stdlib/asyncio/subprocess.pyi

```diff
@@ -65,7 +65,13 @@ if sys.version_info >= (3, 10):
         creationflags: int = ...,
         restore_signals: bool = ...,
         start_new_session: bool = ...,
-        pass_fds: Any = ...,
+        pass_fds: Collection[int] = ...,
+        group: None | str | int = ...,
+        extra_groups: None | Collection[str | int] = ...,
+        user: None | str | int = ...,
+        umask: int = ...,
+        process_group: str | int | None = ...,
```

Comment:
subprocess has this as just `int | None`, which looks correct based on the C code.

Reply:
I dont' understand what you wanted to say.

Reply:
In `subprocess.pyi`, the type of the `process_group` parameter is `int | None`, not `str | int | None`. Based on my reading of the relevant C code, `int | None` appears to be correct.

Reply:
@socketpair, you haven't addressed this comment here. Jelle is asking you to change the annotation for the `process_group` parameter (both for this function and all other functions that have this parameter) to be `int | None` instead of `str | int | None`. You need to either make the change or explain why you think that this would be incorrect.

Reply:
Really. it's my bug. Will fix in next commit.